src/components/Nav/index.tsx

```diff
@@ -42,11 +32,3 @@ export const Nav = () => {
     </div>
   )
 };
-
-// stage changes
-// commit message
```

Comment:
@ContonJon maybe save this somewhere personal so we can prevent too many comments in the code